physics.cc

```diff
@@ -1157,7 +1157,7 @@ void PScene::raycast(float *origin, float *direction, float *meshPosition, float
   }
 }
 
-void PScene::overlap(PxGeometry *geom, float *position, float *quaternion, float *meshPosition, float *meshQuaternion, unsigned int &hit, unsigned int &id) {
+float *PScene::overlap(PxGeometry *geom, float *position, float *quaternion, float *meshPosition, float *meshQuaternion, unsigned int &hit, unsigned int &id) {
```

Comment:
If you are not returning id (returning buffer instead) then I recommend getting rid of the `unsigned int &hit, unsigned int &id`, because it's redundant?

You can pass back `0` address if it's a null hit and detect that case on the JS side.

Reply:
Deleted.